### DISCUSSION TITLE
short between pin 5 and 4 ( s21)

### DISCUSSION THREAD

**Discussion by GL640** [revk/xESP32-Faikout#376] [Q&A]
so as topic title i shorted the ground ( pin5 ) and pin4 ( 12volt ) now i dont have 12 volt on pin 5 anymore ( it seems )
the ac still seems to work fine. does anyone now what is just after pin4 maybe a resistor that could have blown? cant seems to see anythng.
thx!

edit
i still have 5 volt between pin1 and pin5 so those are ok.

edit

i have teste the ac with cooling and heating alsow the wifi app acces ( from daikin cloud ) and it al seems to work.
not sure if that 12 volt is only used for the s21 protocol and deviced connected to it or not.
i dot think there is a fure i think its even strange that the short moment i touched those 2 it could have done anyting.
normaly you just have a little spart and if thats just for a sec nothing really is damaged but here it seems to have.

**Comment by revk** [comment]
If you have shorted the supply from the aircon, it is possible it has some sort of fuse. But I have no idea. Is is a bit sloppy these days to have a fuse though.

**Reply by GL640** [reply]
well i have tested the unit with cooling and heating and the cloud wifi etc. it all works
so the 12 volt is not that far damaged that the ac is no longer working
maybe the 12 volt for the s21 is someting isolated? and that is now damaged
a fuse? well where does that have to be

**Reply by GL640** [reply]
> If you have shorted the supply from the aircon, it is possible it has some sort of fuse. But I have no idea. Is is a bit sloppy these days to have a fuse though.

can i use the 5 volt from pin 1 to power the faikin?

**Reply by revk** [reply]
Unlikely to have enough power

**Reply by GL640** [reply]
> Unlikely to have enough power
maybe any other socket where there is 12 volt for the faikin? i can put a dupont in and get enough? s401 os s402? not sure its a 2 pin underneeth the s21. is that 12volt?

**Reply by GL640** [reply]
> Unlikely to have enough power

would it damage the daikin ? or wil the 5 volt just be there but the faikin will not boot correctly?
i already have the faikin setup so it wil not need to do any first boot setuo, maybe that wil lower the max watt? ( no local wifi? or is that always on ? )

**Comment by GL640** [comment]
i have teste the ac with cooling and heating alsow the wifi app acces ( from daikin cloud ) and it al seems to work.
not sure if that 12 volt is only used for the s21 protocol and deviced connected to it or not.
i dot think there is a fure i think its even strange that the short moment i touched those 2 it could have done anyting.
normaly you just have a little spart and if thats just for a sec nothing really is damaged but here it seems to have.
